js/BookmarkingModel.js

```diff
@@ -0,0 +1,3 @@
+export default class BookmarkingModel extends Backbone.Model {
+

```

Comment:
did you want to add aria label and label defaults?

```suggestion

  defaults() {
    return {
      label: '',
      ariaLabel: ''
    };
  }

```

Reply:
Agreed. Do you suppose it might be better to link this to either a new "_button" or "_global" set of fields in course.json?

Reply:
globals is good

Reply:
I will make an alternate ticket to change the core schemas. But adding this in as well.